umm-spec-lib/src/cmr/umm_spec/umm_g/tiling_system.clj

```diff
@@ -0,0 +1,32 @@
+(ns cmr.umm-spec.umm-g.tiling-system
+  "Contains functions for parsing UMM-G JSON TilingIdentificationSystem into umm-lib granule model
+   TwoDCoordinateSystem and generating UMM-G JSON TilingIdentificationSystem
+   from umm-lib granule model TwoDCoordinateSystem."
+  (:require
+   [cmr.umm.umm-granule :as g])
+  (:import cmr.umm.umm_granule.UmmGranule))
+
+(defn umm-g-tiling-identification-system->TwoDCoordinateSystem
+  "Returns the umm-lib granule model TwoDCoordinateSystem from the given UMM-G TilingIdentificationSystem."
+  [tiling-identification-system]
+  (when tiling-identification-system
+    (let [{:keys [TilingIdentificationSystemName Coordinate1 Coordinate2]}
+          tiling-identification-system]
+      (g/map->TwoDCoordinateSystem
+        {:name TilingIdentificationSystemName
+         :start-coordinate-1 (:MinimumValue Coordinate1)
+         :end-coordinate-1 (:MaximumValue Coordinate1)
+         :start-coordinate-2 (:MinimumValue Coordinate2)
+         :end-coordinate-2 (:MaximumValue Coordinate2)}))))
+
+(defn TwoDCoordinateSystem->umm-g-tiling-identification-system
+  "Returns the UMM-G TilingIdentificationSystem from the given umm-lib granule model TwoDCoordinateSystem."
+  [two-d-coordinate-system]
+  (when two-d-coordinate-system
+    (let [{:keys [name start-coordinate-1 end-coordinate-1 start-coordinate-2 end-coordinate-2]}
+          two-d-coordinate-system]
+      {:TilingIdentificationSystemName name
```

Comment:
What happens when the umm-lib TwoDCoordinateSystemName is not in the defined ENUMs of UMM-G?

Reply:
when/if it is validated against the umm-g json schema, there will be an error.

Reply:
I think we want to map the name to a default value in the defined ENUM list. A valid ECHO10 granule with TwoDCoordinateSystem should not become invalid when it is converted into UMM-G format.